tools.tf

```
@@ -0,0 +1,18 @@
+module "orb-terraform" {
+  source        = "./modules/repository"
+  name          = "orb-terraform"
+  description   = "Development repository for the orb-terraform cookbook"
+  cookbook_team = github_team.orb-terraform.id
```

```suggestion
    status_checks = ["ci/circleci: orb-lint"]
    cookbook_team = github_team.orb-terraform.id
```
Need to setup `status_checks` correctly